packages/pyxis-react/src/components/FormGrid/FormGrid.tsx

```diff
@@ -1,61 +1,43 @@
+import Grid from 'components/Grid';
 import { FC, ReactElement } from 'react';
-import classNames from 'classnames';
-import Row, { RowProps } from './Row';
-import Col, { ColProps } from './Col';
-
-const getClasses = (rowGap: RowGap, className?: string): string => classNames(
-  'grid',
-  { 'grid--gap-large': rowGap === 'large' },
-  className,
-);
 
 const FormGrid: FC<FormGridProps> & FormGridChildren = ({
   children,
-  className,
-  rowGap = 'large',
-  id,
   ...restProps
 }) => (
-  <div className={getClasses(rowGap, className)} data-testid={id} id={id} {...restProps}>
+  <Grid {...restProps}>
     {children}
-  </div>
+  </Grid>
 );
 
 const Header: FC<ChildProps> = ({ children }) => <>{children}</>;
 const Content: FC<ChildProps> = ({ children }) => (
-  <FormGrid.Row>
-    <FormGrid.Col>
-      <div className={getClasses('medium')}>
+  <Grid.Row>
+    <Grid.Col>
+      <Grid>
```

Comment:
The rowgap for fields is missing
```suggestion
      <Grid rowGap='medium'>
```